addon/templates/project/errors.hbs

```diff
@@ -0,0 +1,150 @@
+{{#if
+  (or
+    this.fetchProject.isRunning
+    this.fetchBuildings.isRunning
+    this.fetchEntrances.isRunning
+  )
+}}
+  <div class="uk-flex uk-flex-center uk-flex-middle uk-margin-large-top">
+    <UkSpinner @ratio="3" />
+  </div>
+{{else}}
+  {{#if this.totalErrorCount}}
+    <div uk-alert class="uk-alert-danger">
+      {{t "ember-gwr.issues.infoAlert" count=this.totalErrorCount}}
+    </div>
+  {{else}}
+    <div uk-alert class="uk-alert-success">
+      {{t "ember-gwr.issues.infoSuccess"}}
+    </div>
+  {{/if}}
+  <h3 class="uk-margin-top">
+    {{t "ember-gwr.models.project" count=1}}
+  </h3>
+  <ModelErrors
+    @model={{this.project}}
+    @modelId={{this.project.EPROID}}
+    @linkRoute={{"project.form"}}
```

Comment:
`string` needs no `{{ }}` in template. You can simply assign the value like a normal html attribute `@linkRoute="project.form"`.